package.json

```diff
@@ -101,21 +101,23 @@
 		"@babel/eslint-parser": "^7.13.14",
 		"@babel/eslint-plugin": "^7.13.16",
 		"@babel/preset-react": "^7.13.13",
```

Comment:
Dan is deze en die andere preset die je uit .babelrc hebt verwijderd nu toch niet meer nodig? Of gebruik je die wel nog ergens?